### PR TITLE
Add Natron v2.3.14

### PR DESCRIPTION
**Pull request by luzpaz**
After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].
- [x] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

**Comment by luzpaz** [comment]
Cask was deleted because it was pointing to old website pre-fork.  
ref: https://github.com/Homebrew/homebrew-cask/pull/67251  
I'm also exploring adding this to cask-versions because the beta releases are more frequent then stable.

File: Casks/natron.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'natron' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/natron.rb]: Needs to be set.

File: Casks/natron.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/natron.rb]: Needs to be set.

File: Casks/natron.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+  url 'https://github.com/NatronGitHub/Natron/releases/latest/download/Natron-2.3.14.dmg'
```
Comment (vitorgalvao) [inline Casks/natron.rb]:
```suggestion
  url "https://github.com/NatronGitHub/Natron/releases/latest/download/Natron-#{version}.dmg"
```

Comment (vitorgalvao) [reply Casks/natron.rb]: Needs a comment above (`style` or `audit` will point it out).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
Thanks. Should be good to go

**Pull request merged**